docs/configuration/git-providers/monitoring-conf/prometheus-operator-values.yaml

```diff
@@ -0,0 +1,51 @@
+grafana:
+  enabled: true
+  adminPassword: <--- admin user password --->
+  service:
+    type: NodePort
+    nodePort: 30080
+  sidecar:
+    dashboards:
+      searchNamespace: ALL
+    datasources:
+      defaultDatasourceEnabled: false
+  additionalDataSources:
+    - name: Loki
+      type: loki
+      url: http://loki:3100/
+      version: 1
+      orgId: 1
+      access: proxy
+      basicAuth: false
+      editable: true
+      jsonData:
+         tlsSkipVerify: true
+    - name: Prometheus
+      type: prometheus
+      url: http://prometheus-prometheus-oper-prometheus:9090/
```

Comment:
Doesn't this depend on the namespace?

Reply:
Resolution is made in the same namespace where grafana is running, so no.